Collapsible/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react'
+import {Motion, spring} from 'react-motion'
+import debounce from '../lib/debounce'
+
+export default class Collapsible extends Component {
+  constructor () {
+    super()
+
+    this.state = {
+      height: 0
+    }
+  }
+
+  componentDidMount () {
+    this.debouncedResizeHandler = debounce(() => this.onResize())
+    window.addEventListener('resize', this.debouncedResizeHandler)
```

Comment:
Yes. The installments component, for example, becomes a vertical list rather than a horizontal list at a certain breakpoint. But it could also just be as simple as a line of text breaking into two lines.